src/main/scala/scala/tools/refactoring/analysis/CompilationUnitDependencies.scala

```diff
@@ -310,10 +329,10 @@ trait CompilationUnitDependencies extends CompilerApiExtensions with ScalaVersio
             case Import(select @ Select(Ident(nme.scala_), `language`), feature) =>
               feature foreach (selector => addToResult(Select(select, selector.name)))
 
-            case imp @ Import(qualifier, selector)  => {
+            case imp @ Import(qualifier, selector) => {
               if (inScopeForLocalImports) {
                 if (!qualifier.symbol.isLocal && !qualifier.symbol.isVal) {
-                  if (isRelativeToTopLevelImports(qualifier) || isRelativeToEnclosingPackage(qualifier)) {
+                  if (if (newWay) false else isRelativeToTopLevelImports(qualifier) || isRelativeToEnclosingPackage(qualifier)) {
```

Comment:
I think it is more readable than `!newWay && isRelativeToTopLevelImports(qualifier)`. `newWay` just shows me pieces of code for modification.